### ISSUE TITLE
Unable to install package `gulp-responsive`

### ISSUE THREAD

**Issue by mahnunchik** [npm/npm#8354]
Unable to install package [`gulp-responsive`](https://www.npmjs.com/package/gulp-responsive)

``` bash
$ npm install gulp-responsive
npm ERR! fetch failed https://registry.npmjs.org/gulp-responsive/-/gulp-responsive-1.1.0.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed https://registry.npmjs.org/gulp-responsive/-/gulp-responsive-1.1.0.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
and so on...
```

``` bash
$ npm info gulp-responsive

{ name: 'gulp-responsive',
  description: 'gulp responsive images',
  'dist-tags': { latest: '1.1.0' },
  versions: 
   [ '0.0.1',
     '1.0.0',
     '1.0.1',
     '1.0.2',
     '1.0.3',
     '1.0.4',
     '1.0.5',
     '1.0.6',
     '1.0.7',
     '1.1.0' ],
  maintainers: 'mahnunchik <mahnunchik@gmail.com>',
  time: 
   { modified: '2015-05-24T08:10:20.351Z',
     created: '2014-10-26T18:45:44.853Z',
     '0.0.1': '2014-10-26T18:45:44.853Z',
     '1.0.0': '2014-10-29T05:44:09.060Z',
     '1.0.1': '2014-11-03T10:29:33.962Z',
     '1.0.2': '2014-11-11T18:11:34.232Z',
     '1.0.3': '2014-11-18T19:33:10.463Z',
     '1.0.4': '2014-11-22T21:15:30.148Z',
     '1.0.5': '2015-01-28T08:13:51.910Z',
     '1.0.6': '2015-02-08T08:08:51.221Z',
     '1.0.7': '2015-02-16T17:32:35.098Z',
     '1.1.0': '2015-05-24T08:10:20.351Z' },
  homepage: 'https://github.com/mahnunchik/gulp-responsive#readme',
  keywords: 
   [ 'gulpplugin',
     'gulp',
     'sharp',
     'responsive',
     'img',
     'images',
     'picture',
     'picturefill',
     'scrset' ],
  repository: 
   { type: 'git',
     url: 'git+https://github.com/mahnunchik/gulp-responsive.git' },
  author: 'Evgeny Vlasenko <mahnunchik@gmail.com> (https://github.com/mahnunchik)',
  bugs: { url: 'https://github.com/mahnunchik/gulp-responsive/issues' },
  license: 'MIT',
  readmeFilename: 'README.md',
  users: { owanturist: true, mahnunchik: true },
  version: '1.1.0',
  main: 'lib',
  engines: { node: '>=0.10.0' },
  scripts: { test: 'npm run lint && mocha', lint: 'jshint lib test' },
  dependencies: 
   { async: '^1.0.0',
     'gulp-util': '^3.0.1',
     lodash: '^3.0.0',
     minimatch: '^2.0.1',
     rename: '^1.0.3',
     sharp: '^0.10.0',
     through2: '^0.6.3' },
  devDependencies: { jshint: '^2.5.10', mocha: '^2.0.1' },
  gitHead: '365e5d17699ebc01228b79d0551c07a17d3c95bb',
  dist: 
   { shasum: '43d9ff0d051325c7e362b2d02f8973371f0a883a',
     tarball: 'http://registry.npmjs.org/gulp-responsive/-/gulp-responsive-1.1.0.tgz' },
  directories: {} }
```

Following URl returns 404 http://registry.npmjs.org/gulp-responsive/-/gulp-responsive-1.1.0.tgz

**Comment by mahnunchik** [comment]
System info:
```bash
$ node -v
v2.1.0
$ npm -v
2.10.1
```

**Comment by astorije** [comment]
I have a similar issue for [chai-immutable](https://github.com/astorije/chai-immutable) as discussed [here](https://github.com/astorije/chai-immutable/issues/15). Version 1.0.0 is broken while version 0.3.0 was successfully published earlier.

FYI, the publication process I used was to run `npm version 1.0.0 && git push --tags` and then let Travis CI do the deployment. This can be seen in the [.travis.yml](https://github.com/astorije/chai-immutable/blob/046dd4ff9d8ad96eaa572c5dc16656d6a62de458/.travis.yml#L8). Both 1.0.0 and 0.3.0 versions were published with the exact same way.

Any idea?

**Comment by mahnunchik** [comment]
Hi @astorije 
I also used Travis CI for deployment.

**Comment by astorije** [comment]
As @ceejbot [mentioned](https://github.com/astorije/chai-immutable/issues/15#issuecomment-109091385), that was a bug on npm that was fixed after that \o/

**Comment by mahnunchik** [comment]
Thanks @astorije 

I unpublished `gulp-responsive@1.1.0`

**Comment by iarna** [comment]
@astorije's problem was that the version of node that published it was `0.11.16`, which is broken and can't publish valid npm packages (a broken `crypto.Hash` implementation).
